src/options.cpp

```diff
@@ -99,6 +99,10 @@ class arg_parser {
       this->is_ignoring_options_ = true;
       this->option_ = std::nullopt;
     } else if (this->current_arg()[0] == '-') {
+      if (std::strcmp(this->current_arg(), "-") == 0) {
```

Comment:
Nit: `this->current_arg()` is a string_view, so we can compare with `==`:
```suggestion
      if (this->current_arg() == "-"sv) {
```